### ISSUE TITLE
Can't install spotify on ubunto distro 21.10

### ISSUE THREAD

**Issue by CarlosEmanuelMartins** [abba23/spotify-adblock#58]
spotify only makes the repo work with the stable release of the ubunto currently 20.04 anything above will not work.
tring to install spotify from the recomened link will give this error:
```
$ sudo apt-get update && sudo apt-get install spotify-client

Ign:1 http://ppa.launchpad.net/dawidd0811/neofetch/ubuntu impish InRelease                                             
Get:2 http://security.ubuntu.com/ubuntu impish-security InRelease [110 kB]                                             
Err:3 http://ppa.launchpad.net/dawidd0811/neofetch/ubuntu impish Release                                               
  404  Not Found [IP: 91.189.95.85 80]
Hit:4 https://dl.google.com/linux/chrome/deb stable InRelease                                                          
Hit:5 http://pt.archive.ubuntu.com/ubuntu impish InRelease                                                             
Hit:6 http://repository.spotify.com stable InRelease                                                                   
Hit:7 http://pt.archive.ubuntu.com/ubuntu impish-updates InRelease                                                     
Hit:8 http://pt.archive.ubuntu.com/ubuntu impish-backports InRelease                                                   
Reading package lists... Done                                                                                          
E: The repository 'http://ppa.launchpad.net/dawidd0811/neofetch/ubuntu impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

The command `make` will give this error:

```
# cargo build --profile release
cargo build --release
make: cargo: No such file or directory
make: *** [Makefile:14: target/release/libspotifyadblock.so] Error 127
```

**Comment by abba23** [comment]
>spotify only makes the repo work with the stable release of the ubunto currently 20.04 anything above will not work.

I don't think I can help if that's the case. That said, I don't see an error related to Spotify in that log. The update that's failing seems to be "neofetch".


>`cargo: No such file or directory`

That usually means that you forgot to install Rust.

**Comment by CarlosEmanuelMartins** [comment]
> That usually means that you forgot to install Rust.
I Positive I installed rust.

Read.me recommends installing spotify from this page: [www.spotify.com/us/download/linux](https://www.spotify.com/us/download/linux/]) ware its stated.
> Spotify for Linux is also released as a Debian package. **Our aim is that it should work with the latest Long Term Support release of Ubuntu,** but we will try to make it work for other releases of Ubuntu and Debian as well.

In the above error message I runned the command to install spotify and update/upgrade command at the same time and I guess the operation stopped at the upgrade or update and I didn't notice it so the error is not relevant. I'm sorry I was sleepy and tired spent the entire day tring to make it work.

I don't have the exact error message as I have installed ubunto 20.4 again but paraphrasing  **the relevant error message was:**
`"W: The repository 'http://repository.spotify.com stable non-free' does not have a Release file."` 

This [stack exange answare](https://askubuntu.com/questions/866901/what-can-i-do-if-a-repository-ppa-does-not-have-a-release-file) explains this warning to mean:
> The PPA you've added does not support your version of Ubuntu, meaning there aren't any packages for your particular release...

I'm not sure if there was a way to try to install the old release in the new one but I ddint no find any simple way to do and so decided to install ubunto 20.04.
